content/200-concepts/300-more/100-editor-setup.mdx

```diff
@@ -14,6 +15,8 @@ If you don't see your editor here, please [open a feature request](https://githu
 
 You can install the [Prisma VS Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma).
 
-## Emacs
+## Community projects
+
```

Comment:
Maybe a disclaimer here that these are not offered by us, might not have the same or complete feature set and so on.